packages/fxa-settings/e2e/tools/firefox_profile.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const path = require('path');
+const childProcess = require('child_process');
+
+module.exports = function createProfile(config) {
+  console.log('Creating Firefox profile...');
+
+  let encodedProfile = '';
+  const profileArgs = JSON.stringify(JSON.stringify(config));
+  const profileTool = path.join('e2e', 'tools', 'firefox_profile_creator.js');
+  try {
+    encodedProfile = childProcess.execSync(
+      ['node', profileTool, profileArgs].join(' ')
+    );
+  } catch (e) {
+    console.log('Note: execSync failed to run:', e);
```

Comment:
Assuming setting this is critical to test run, shouldn't we let this exception happen (instead of just console.logging)?